CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Allow to configure `cgroups` v1 or v2 compatibility per node pool, instead of the whole cluster.
+- Control plane nodes always use cgroups v2.
```

Comment:
Why a new line here? Wouldn't it make more sense to have it in the previous line?